### PR TITLE
Pull docker-compose images before running them

### PR DESCRIPTION
**Pull request by BenediktSeidl**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ jobs:
           path: dist
 
       - name: run the e2e tests
-        run: docker-compose up --exit-code-from=cypress --pull always
+        run: docker-compose pull && docker-compose up --exit-code-from=cypress
```
Comment (PhilippSiegmantel) [inline .github/workflows/ci.yml:51]: The internet says that we should put a `docker-compose rm -f ` in front of that to make sure no cached images are used. `docker-compose pull` alone can still use stuff that is already there, if I remember correctly.

**Review by PhilippSiegmantel — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (BenediktSeidl) [reply .github/workflows/ci.yml:51]: before adding this command i would like to fully understand the problem. do you have a description of the problem, and how `docker-compose rm -f` solves this problem?
(currently no e2e tests run in the main branch, maybe we should merge this as is, and discuss this in another channel?)

Comment (PhilippSiegmantel) [reply .github/workflows/ci.yml:51]: I once had the issue, that a change wasn't deployed through docker even though a new image was build, available in the registry, and `docker-compose pull` was called explicitly. That's when I found out that docker(-compose) doesn't always check if a newer image version is available when it has a local version of that image. Only way to force it to do so is to remove the image locally and then pull or use a different tag each time, which I think you are supposed to.

Comment (BenediktSeidl) [reply .github/workflows/ci.yml]: okay, i looked into it. `docker-compose` seems to be discontinued and replaced by `docker compose`. you can install it via `apt-get install docker-compose-plugin`. i would hope that this means, that we don't have to delete the images?

Comment (PhilippSiegmantel) [reply .github/workflows/ci.yml]: I'll try it out, but let's hope it does.

**Review by PhilippSiegmantel — APPROVED** [review]

**Pull request merged**